angular/angular-cli
# Add prod polyfills

---
### Overview
Add support for minified versions of the polyfills.

### Changes
* Updated `index.html` blueprints to include a prodPolyfills repeater and a polyfills repeater that switches based on the environment.
* Updated `angular-cli-build.js` blueprint patterns for polyfills to copy over both the prod & non-prod polyfills.
* Updated `angular2-app.js` to include both the prodPolyfills and polyfills scripts 


### Additional Notes
* A minified bundle of the polyfills would be a nice next step

### Testing Instructions
 1. using the blueprint version of the html file
 2. run `ng build -prod` and notice the minified versions of the polyfills will be referenced in the _comipled_ `index.html`
 3. run `ng build` and noticce the non-minified versions of the polyfills will be referenced in the _compiled_ `index.html`
